hokusai/commands/init.py

```diff
@@ -0,0 +1,169 @@
+import os
+
+from distutils.dir_util import mkpath
+from collections import OrderedDict
+
+import yaml
+
+from jinja2 import Environment, PackageLoader
+env = Environment(loader=PackageLoader('hokusai', 'templates'))
+
+from hokusai.config import HokusaiConfig
+from hokusai.common import print_green, build_service, build_deployment, YAML_HEADER
+
+def init(project_name, aws_account_id, aws_ecr_region, framework, base_image,
+          run_command, development_command, test_command, port, target_port,
+            with_memcached, with_redis, with_mongo, with_postgres):
```

Comment:
Right - the init command still bothers me.  It could me made interactive as it's only meant to be run when adding hokusai to a project?